libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/listener/OracleModeSqlParserListener.java

```diff
@@ -700,4 +700,62 @@ public void enterRollback_stmt(Rollback_stmtContext ctx) {
         this.dbObjectType = DBObjectType.OTHERS;
     }
 
+    @Override
+    public void enterSet_comment_stmt(OBParser.Set_comment_stmtContext ctx) {
+        // comment on xxx
+        setSqlType(SqlType.COMMENT);
+        ParseTree child = null;
+        if (ctx.getChildCount() >= 3) {
+            child = ctx.getChild(2);
+        }
+        String sqlType = child != null ? child.getText() : "";
+        switch (sqlType) {
+            case "TABLE":
+                this.dbObjectType = DBObjectType.TABLE;
+                break;
+            case "COLUMN":
+                this.dbObjectType = DBObjectType.COLUMN;
+                break;
+            case "VIEW":
```

Comment:
```
set_comment_stmt
    : COMMENT ON TABLE normal_relation_factor IS STRING_VALUE
    | COMMENT ON COLUMN column_definition_ref IS STRING_VALUE
    ;
```

we just support set comment on table and column~ why should we support so many objects?

Reply:
ok

Reply:
comment on table xxx is "XXX" dont enter `set_comment_stmt` method